static/styles/settings.scss

```diff
@@ -577,8 +580,6 @@ input[type=checkbox].inline-block {
         float: none;
     }
 
-    #organization .settings-section .admin-realm-form .controls,
-    #settings .settings-section .account-settings-form .controls,
```

Comment:
This removed line doesn't match the pattern of the rest -- what's it's story?

Reply:
Removed #settings line part by mistake in this commit, these needed to be removed in other commit, changed and updated it. Will proofread the changes carefully from the next time.